scrapers/teiid.py

```diff
@@ -94,6 +96,17 @@ def query(self, sql, db='public', retry=None):
         if retry is not None and retry < 1:
             raise ValueError('The retry keyword must contain a value greater than 0')
 
+        one_min_ago = datetime.utcnow() - timedelta(mins=1)
```

Comment:
I am trying to test this and I am getting a traceback:

Traceback (most recent call last):
  File "scripts/scrape.py", line 67, in <module>
    scraper.run(since=since)
  File "/home/ynanavat/Documents/Git-repos/purview-api/scrapers/koji.py", line 32, in run
    builds = self.get_koji_builds(since)
  File "/home/ynanavat/Documents/Git-repos/purview-api/scrapers/koji.py", line 70, in get_koji_builds
    return self.teiid.query(sql=sql_query)
  File "/home/ynanavat/Documents/Git-repos/purview-api/scrapers/teiid.py", line 99, in query
    one_min_ago = datetime.utcnow() - timedelta(mins=1)
TypeError: 'mins' is an invalid keyword argument for this function

Reply:
Thanks, I made a typo and fixed it.